docs/templates/index.md

```diff
@@ -0,0 +1,199 @@
+| ImageID           | string            | The ID of the Package to use when launching a Template.                       | 
+| FirewallEnabled   | bool              | Enable or disable the Firewall on the instances launched. Default is `false`. | 
+| Networks          | []string          | A list of network IDs to attach to the instances launched.                    | 
+| UserData          | string            | Data to be copied to the instances on boot.                                   | 
+| MetaData          | map[string]string | A mapping of metadata to apply to the instances launched.                     | 
+| Tags              | map[string]string | A mapping of tags to apply to the instances launched.                         |  
+
+### GET /v1/tsg/templates
+
+To list all of the templates associated with a specific Triton account, send a `GET` request to `/v1/tsg/templates` with
+the request headers detailed below.
+
+#### Example Request
+
+```
+curl -X GET "https://tsg.us-sw-1.svc.joyent.zone/v1/tsg/templates"
+```
+
+
+#### Request Headers 
+
+```
+Authorization: Signature keyId="/test-user-name/keys/aa:bb:cc:dd:9c:54:e9:78:3f:80:0d:ba:6b:c6:ff:44",algorithm="rsa-sha1",headers="date",signature="..."
+Date: Fri, 06 Apr 2018 18:33:38 UTC
+```
+
+#### Sample Response
+
+```
+[
+    {
+        "id": "437c560d-b1a9-4dae-b3b3-6dbabb7d23a7",
+        "template_name": "test-template-6",
+        "account_id": "6f873d02-172c-418f-8416-4da2b50d5c53",
+        "package": "test-package",
+        "image_id": "49b22aec-0c8a-11e6-8807-a3eb4db576ba",
+        "firewall_enabled": false,
+        "networks": [
+            "f7ed95d3-faaf-43ef-9346-15644403b963"
+        ],
+        "metadata": {
+    	    "root_pw": "s8v9kuht5e"
+    	}, 
+        "tags": {
+            "role": "web",
+            "owner": "api-team"
+        }
+    }
+}
+```
+
+### GET /v1/tsg/templates/{identifier}
+
+To show information about a specific template, send a `GET` request to `/v1/tsg/templates/{identifier}` 
+using the request headers as detailed below.
+ 
+
+#### Example Request
+
+```
+curl -X GET "https://tsg.us-sw-1.svc.joyent.zone/v1/tsg/templates/319209784155176962"
+```
+
+#### Request Headers 
+
+```
+Authorization: Signature keyId="/test-user-name/keys/aa:bb:cc:dd:9c:54:e9:78:3f:80:0d:ba:6b:c6:ff:44",algorithm="rsa-sha1",headers="date",signature="..."
+Date: Fri, 06 Apr 2018 18:33:38 UTC
+```
+
+#### Sample Response
+
+```
+{
+    "id": "437c560d-b1a9-4dae-b3b3-6dbabb7d23a7",
+    "template_name": "test-template-6",
+    "account_id": "6f873d02-172c-418f-8416-4da2b50d5c53",
+    "package": "test-package",
+    "image_id": "49b22aec-0c8a-11e6-8807-a3eb4db576ba",
+    "firewall_enabled": false,
+    "networks": [
+        "f7ed95d3-faaf-43ef-9346-15644403b963"
+    ],
+    "metadata": {
+	    "root_pw": "s8v9kuht5e"
+	}, 
+    "tags": {
+        "role": "web",
+        "owner": "api-team"
+    }
+}
+```
+
+### POST /v1/tsg/templates
```

Comment:
The file directory strings in headers should still be wrapped in backticks:

### POST `/v1/tsg/templates`